README.md

```diff
@@ -21,40 +21,42 @@ Usage: auto-changelog [options]
-      --starting-date [yyyy-mm-dd]    # specify earliest date to include in changelog
-      --ending-version [tag]          # specify latest version to include in changelog
-      --sort-commits [property]       # sort commits by property [relevance, date, date-desc, subject, subject-desc], default: relevance
-      --release-summary               # display tagged commit message body as release summary
-      --unreleased-only               # only output unreleased changes
-      --hide-empty-releases           # hide empty releases
-      --hide-credit                   # hide auto-changelog credit
-      --handlebars-setup [file]       # handlebars setup file
-      --append-git-log [string]       # string to append to git log command
-      --append-git-tag [string]       # string to append to git tag command
-      --prepend                       # prepend changelog to output file
-      --stdout                        # output changelog to stdout
-  -V, --version                       # output the version number
-  -h, --help                          # output usage information
+  -o, --output [file]                   # output file, default: CHANGELOG.md
+  -c, --config [file]                   # config file location, default: .auto-changelog
+  -t, --template [template]             # specify template to use [compact, keepachangelog, json], default: compact
+  -r, --remote [remote]                 # specify git remote to use for links, default: origin
+  -p, --package                         # use version from package.json as latest release
+  -v, --latest-version [version]        # use specified version as latest release
+  -u, --unreleased                      # include section for unreleased changes
+  -l, --commit-limit [count]            # number of commits to display per release, default: 3
+  -b, --backfill-limit [count]          # number of commits to backfill empty releases with, default: 3
+      --commit-url [url]                # override url for commits, use {id} for commit id
+      --issue-url [url]                 # override url for issues, use {id} for issue id
+      --merge-url [url]                 # override url for merges, use {id} for merge id
+      --compare-url [url]               # override url for compares, use {from} and {to} for tags
+      --issue-pattern [regex]           # override regex pattern for issues in commit messages
+      --breaking-pattern [regex]        # regex pattern for breaking change commits
+      --merge-pattern [regex]           # add custom regex pattern for merge commits
+      --ignore-commit-pattern [regex]   # pattern to ignore when parsing commits
+      --include-commit-pattern [regex]  # pattern to include when parsing commits, merge commits are be included by default unless "--strict-include" is checked
+      --strict-include                  # strictly include only commits that match the pattern by --include-commits-pattern
```

Comment:
yea, so when you use the `include pattern`, if you dont put something in your regex to include merge commits and PRs then it will not include them, so what this does is, by default all merge commits are included when you use `include commit`, meaning all merge commits + all the other commits that match `include pattern`. however if you wan to make it strict, meaning strictly include by the provided regex, the. you'd be using that flag

Reply:
so `--strict-include` is sugar for `--merge-pattern=''` or something?

Reply:
so basically, having this lets you not have to include the merge commit regex in your `include pattern` regex if you want merge commits to be part of the included commits

Reply:
> so `--strict-include` is sugar for `--merge-pattern=''` or something?

no, let me explain with an example, I have this as my `include pattern` `/^fix/`.
so what happens now is all the commits that match this will be included, but merge commits if they don't match this wont be included, however you most probably want all the merge commits (PRs) to be included always (with or without custom `mergeCommit` option), 
so by default the `include pattern` will include all the merge commits (with or without custom `mergeCommit` option). but in case you want the including be strict to your provided pattern, you will use `strict-include` flag

Reply:
@ljharb maybe if you think it should be removed and provide an explaination for how to achive a `--strict-include` by doing `--merge-comiit=''` ?

Reply:
That sounds simpler to me.

Reply:
ok fair enough, I'll update it

Reply:
@ljharb well actually that would be a sugar for `no merge commit pattern`, meaning there would be no difference between merge commits and regualr commits

Reply:
I guess i'd assume that the strict thing always applies, and that `include pattern` is a filter for "all the commits", and then whatever's left is what has the normal logic applied?

Reply:
@ljharb what we can probably do is, to make the `getMergePattern()` return `/[]/` in case the `--merge-pattern=''`

Reply:
> I guess i'd assume that the strict thing always applies, and that `include pattern` is a filter for "all the commits", and then whatever's left is what has the normal logic applied?

well imo, PRs and merge commits have higher value, so usually you would want them included, but probably dont want to go through a crazy regex if you want to include certain commits + all merge commits

Reply:
hmm. it seems like the use cases motivating this PR probably needs a bit more fleshing out :-)

it's helpful to think in terms of minimal modification to existing functionality. If it takes two conceptual changes, then it's nice to figure out a way to split them into two separately motivated PRs.

Reply:
maybe a better practice and compromise is to have `--include-commit-pattern` strict by default, and then provide a seperate flag to '--include-merge-commits`

Reply:
> hmm. it seems like the use cases motivating this PR probably needs a bit more fleshing out :-)
> 
> it's helpful to think in terms of minimal modification to existing functionality. If it takes two conceptual changes, then it's nice to figure out a way to split them into two separately motivated PRs.

valid, well let me know what do you think about my last suggestion, and if you'd still prefer to split into another PR for that or not

Reply:
@ljharb thinking now, it would make more sense to split out, because the same situation can apply to `--ignore-commit-pattern` where you may want to ignore commits but still include all marge commits